include/h2o.h

```diff
@@ -673,6 +673,18 @@ typedef struct st_h2o_res_t {
     h2o_mime_attributes_t *mime_attr;
 } h2o_res_t;
 
+
+/**
+ * debug state (currently only for HTTP/2)
+ */
+typedef struct st_h2o_debug_state_t{
+    h2o_iovec_vector_t json;
+    ssize_t conn_flow_in;
+    ssize_t conn_flow_out;
+} h2o_debug_state_t;
+
+h2o_debug_state_t *h2o_http2_get_debug_state(h2o_req_t *req, int hpack_enabled);
```

Comment:
Please move the prototype to http2_internal.h, since the function is going to be referred via callback from outside of the http2 stack.